cloud/functions/src/crawlers/devoxx/crawler.ts

```diff
@@ -169,9 +169,11 @@ const crawlDevoxxDay = async (eventId: string, day: string) => {
                 const scheduleTalk = toScheduleTalk(i)
                 const talk = {
                     ...scheduleTalk, 
+                    start: start as ISODatetime,
+                    end: end as ISODatetime,
                     summary: i.proposal?.summary ?? "",
                     description: i.proposal?.description ?? ""
-                }
+                } as DetailedTalk
```

Comment:
FYI, `as XXX` typescript syntax will not enforce that object previously defined will "match" the structure of `XXX` (I mean, if you make a typo in the `DetailedTalk` structure, it will not be caught with the `as XXX` syntax)
`as XXX` syntax is more like a "type cast" meaning "trust me, the object defined previously is of type XXX" (like cast in Java, that's _generally_ not a good idea to use it, except if you know what you're doing)

If you want to ensure such validation check, you need to simply declare `talk` as being of type `DetailedTalk`, which will be seen differently from Typescript (if you try to put a data structure not following `DetailedTalk` type into `talk`, Typescript compiler will complain)

That's a concern I regularly complain about as I don't see it very natural, particularly for newcomers to the language.